templates/cloud/kubernetes.html

```diff
@@ -0,0 +1,116 @@
+            <p>Even for experts, installing and operating Kubernetes at scale is daunting. By using Ubuntu and Juju, Canonical&rsquo;s Kubernetes is simple, naturally elastic and integrates with common operations dashboards. Charms deliver the latest stable Kubernetes and Juju makes it easy to manage and grow your cluster.</p>
+        </div>
+
+        <div class="box six-col last-col equal-height--item">
+            <h2>Your choice of infrastructure</h2>
+            <p>The Canonical Distribution of Kubernetes is certified on most major public clouds and on-premise private infrastructure like VMware, OpenStack or bare-metal MAAS. The ability to move workloads is critical for organisations to remain competitive, foster innovation and avoid infrastructure lock-in.</p>
+        </div>
+
+        <div class="box six-col equal-height--item">
+            <h2>Analytics on tap</h2>
+            <p>Canonical Kubernetes comes integrated with Prometheus for monitoring, Ceph for storage and a fully integrated Elastic stack including Kibana for analysis and visualisations.</p>
+        </div>
+
+        <div class="box six-col last-col equal-height--item">
+            <h2>Cost-effective at scale</h2>
+            <p>The major cost of Kubernetes is the  operations expertise needed to run at scale.  Canonical&rsquo;s Kubernetes bundle includes all operational scripting and tooling needed to manage a long-lived cluster, including upgrades and elastic scaling. With Juju you get all the industries best practices for Kubernetes ops for free.</p>
+        </div>
+    </div>
+</section>
+
+<section class="row">
+    <div class="six-col">
+        <h2>Production features</h2>
+        <ul class="list-ticks--compact">
+            <li>Access to the latest stable upstream Kubernetes release</li>
+            <li>Access to daily ‘edge’ builds of upstream Kubernetes development branch (1.5.x) with rollback</li>
+            <li>Scale master nodes independently of worker nodes</li>
+            <li>Workloads are automatically portable between public clouds, private clouds and bare metal with an easy onramp to Google Container Engine (GKE)</li>
+            <li>Easily create and remove user access</li>
+            <li>Maintenance mode and supported upgrade path to new upstream versions</li>
+            <li>Kubernetes dashboard, Prometheus, for monitoring and alerting, collection and processing of system metrics</li>
+            <li>Elastic Stack for Insights with Beats log collection and monitoring and analysis and visualisation with Elasticsearch and Kibana</li>
+            <li>Optionally backed by Ceph for filesystem and block storage</li>
+        </ul>
+    </div>
+    <div class="juju-bundle-card six-col last-col">
+        <div class="juju-card" data-id="canonical-kubernetes"></div>
+    </div>
+</section>
+
+<section class="row row-grey">
+    <div class="equal-height">
+        <div class="four-col equal-height__item equal-height__align-vertically">
```

Comment:
I think the not-for-small should be here... not on the image... otherwise there is a gap on small screens